src/libstd/collections/hash/table.rs

```diff
@@ -233,12 +231,14 @@ fn can_alias_safehash_as_hash() {
 }
 
 impl<K, V> RawBucket<K, V> {
-    unsafe fn offset(self, count: isize) -> RawBucket<K, V> {
-        RawBucket {
-            hash: self.hash.offset(count),
-            pair: self.pair.offset(count),
-            _marker: marker::PhantomData,
-        }
+    unsafe fn hash(&self) -> *mut HashUint {
```

Comment:
Could you add a comment noting that these functions are unsafe because you can safely create out-of-bounds buckets?